### DISCUSSION TITLE
i broke leather engine

### DISCUSSION THREAD

**Discussion by ghost** [Leather128/LeathersFunkinEngine#230] [Q&A] [answered]
lets start with the story of how i have broke leather engine 

i always wanted to port ui skins from other mods, so i had three in my mind: neo shaggy, vs fireboy and watergirl and another mod vs baphie, so i went creating the folders and everything and i started with neo shaggy, it was ok because the xml was very easy and i made it, ahead i went to try out the vs baphie but i thought it was very difficult and i left the png and xml on the arrows if i could do thatui skin back, then i tried the fireboy and watergirl noteskin, the xml was different so i have strugled a little to do so, when i finally made it i tried to enter another song but it literally crashes (but the songs were using the normal ui skin) the songs i tried to acess was in one mods folder so i tried another one but it stils crashes( the reason it crashes was because of the vs baphie one) so i had the idea of deleting all the content related from vs baphie on that folder, but after i did this i enter another song and appears this message '[lime.utils.assets] ERROR: there is no TEXT asset with an ID of assets/data/ui skins/baphie/config.txt' , but i have already emptied to the trash with the vs baphie things so i started to get desperate emotionally and phisicaly so the only option i could think was delete the folder for leather i made and download another one, i did but even if there werent any mods i still get that message, i try everything i could to make that message dissapear and nothing worked, im so sad and so dumb for made that act, leather was the best to me program and port whenever songs i want to play better 
 
1 point: the grammar errors i made is because im from another country 
2 point: if you didn't understand some part just tell me and i can resume it better

**Comment by Vortex2Oblivion** [comment, marked as answer]
go to C:\Users\username\AppData\Roaming and delete the folder called "leather128"
